README.md

```diff
@@ -12,7 +12,11 @@
 
 ## Introduction
 
-**nf-core/mcmicro** is a bioinformatics pipeline that ...
+!!! We are currently in the process of porting the original MCMICRO to nf-core. This pipeline is therefore in active development !!!
```

Comment:
```suggestion
:::warning 
We are currently in the process of porting the original MCMICRO to nf-core. This pipeline is therefore in active development.
:::
```
In case you want to have this element styled on the website (phil's suggestion is styling it for github)